sw/device/lib/testing/test_rom/test_rom_start.S

```diff
@@ -170,12 +170,17 @@ _start:
   andi t0, t0, 0x1
   beqz t0, .L_ast_done_loop
 
-  // Enable jittery clock if enabled in OTP.
+  // Enable jittery clock if not strictly disabled in OTP.
+  // Use hardcoded MULTIBIT_ASM_BOOL4_TRUE as the enable word to reduce all
```

Comment:
Do you mean "Use MULTIBIT_ASM_BOOL4_FALSE as the disable word to reduce all other possible OTP values to enable."

Reply:
I meant to use the `MULTIBIT_ASM_BOOL4_TRUE` word as the write value to reduce all other options to True (Enable).